AppCenter/AppCenter/MSConstants.h

```diff
@@ -63,3 +63,9 @@ typedef NS_ENUM(NSInteger, MSInitializationPriority) {
   MSInitializationPriorityHigh = 750,
   MSInitializationPriorityMax = 999
 };
+
+typedef NS_OPTIONS(NSUInteger, MSFlags) {
```

Comment:
So we decided to go with MSFlags for all app center flags? My only concern is people won't know which flag works with which API without looking at the documentation. But it's your call.